src/content/docs/pt-br/basics/astro-components.mdx

```diff
@@ -91,12 +91,12 @@ const { titulo } = Astro.props;
 </ul>
 
 <!-- Use uma diretiva de template para construir nomes de classe a partir de múltiplas strings ou até mesmo objetos! -->
-<p class:list={["adicione", "dinâmicos", {classNames: true}] />
+<p class:list={["adicione", "dinâmicos", {classNames: true}]} />
 ```
 
 ## Design baseado em componente
 
-Componentes são projetados para serem **reusáveis** e **combináveis**. Você pode utilizar componentes dentro de outros componentes, criando UIs cada vez mais avançadas. Por exemplo, um componente `Botao` pode ser utilizado para criar um componente `GrupoBotoes` assim:
+Componentes são projetados para serem **reutilizáveis** e **combináveis**. Você pode utilizar componentes dentro de outros componentes para criar UIs cada vez mais avançadas. Por exemplo, um componente `Botao` pode ser utilizado para criar um componente `GrupoBotoes`:
```

Comment:
Going for "reutilizável" as a better translation for "reusable"; also making the translation more in line with the original.